shared/lib/translator.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from flask import current_app
+from langdetect import detect as lang_detect
+import requests
+
+_API_URL = "https://translation.googleapis.com/language/translate/v2"
+_API_HEADER = {'content-type': 'application/json'}
+_EN_LANG = "en"
+
+
+# Detects the query language and translates non-English queries to English.
+# TODO: Add tests once I understand out how to access the api key in tests.
```

Comment:
If we want to test the actual behavior of the translate API, then this should be covered by a golden test?  If you add one golden integration test now with other language, i would expect to work since integration test can find the api key from gcp now: https://github.com/datacommonsorg/website/blob/master/server/app_env/integration_test.py#L21